owasp-suppressions.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress>
+    <notes><![CDATA[
+   Any hypertrace dep
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.hypertrace\..*@.*$</packageUrl>
+    <cpe>cpe:/a:grpc:grpc</cpe>
+    <cpe>cpe:/a:utils_project:utils</cpe>
+  </suppress>
+
+  <!-- This is the final version, and its coming via transitive dep. -->
+  <suppress>
+    <notes><![CDATA[
+    We need to get pinot to upgrade this dep, it's 16 years old. Upgrades fix this issue but have changed package/group names
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/commons\-httpclient/commons\-httpclient@.*$</packageUrl>
+    <cve>CVE-2012-5783</cve>
+    <cve>CVE-2020-13956</cve>
+  </suppress>
+
+  <!-- This is the latest version -->
+  <suppress>
+    <notes><![CDATA[
+   file name: zookeeper-api-1.2.0.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.helix/zookeeper\-api@.*$</packageUrl>
+    <cpe>cpe:/a:apache:zookeeper</cpe>
+    <cpe>cpe:/a:apache:helix</cpe>
+  </suppress>
+
+  <!-- This is the latest version -->
+  <suppress>
```

Comment:
We're constraining janino I thought? If I recall janino had the same disputed issue as jackson too, and that would be suppressed with a temporary CVE suppression but a CPE suppression is only used when the vulnerability is actually referring to a different package.

Reply:
Done